### ISSUE TITLE
Cant enable and find core dumped

### ISSUE THREAD

**Issue by vitassuper** [docker-library/python#880]
When my Python program fails in a Docker container, no core dump or backtrace is generated. I tried enabling it using `ulimit -c unlimited`, but it didn't help. I cannot find a stack trace to determine why my program is crashing. All I get is a `corrupted size vs. prev_size` message.

**Comment by LaurentGoderre** [comment]
Do you have an example that we can take a look at?

**Comment by vitassuper** [comment]
> Do you have an example that we can take a look at?

Here is my program https://github.com/vitassuper/flux-bot and screen below
<img width="577" alt="image" src="https://github.com/docker-library/python/assets/37179932/1515718e-e96d-413e-9909-938670add63a">

**Comment by LaurentGoderre** [comment]
It would be helpful to add env vars to your docker compose to get it running quicky

```yaml
environment:
      - DEBUG_MODE=false
      - SQLALCHEMY_DATABASE_URI=postgres://root:root@postgres
      - SQLALCHEMY_DATABASE_DB=connector
      - POSTGRES_SERVER=postgres
      - POSTGRES_USER=root
      - POSTGRES_PASSWORD=root
      - POSTGRES_DB=trading_view_connector
      - APP_KEY=1234
      - TELEGRAM_BOT_TOKEN=1234
      - API_PASSWORD=1234
      - TELEGRAM_CHAT_ID=1234
      - TELEGRAM_CHAT_ID2=1234
```

**Comment by vitassuper** [comment]
> It would be helpful to add env vars to your docker compose to get it running quicky
> 
> ```yaml
> environment:
>       - DEBUG_MODE=false
>       - SQLALCHEMY_DATABASE_URI=postgres://root:root@postgres
>       - SQLALCHEMY_DATABASE_DB=connector
>       - POSTGRES_SERVER=postgres
>       - POSTGRES_USER=root
>       - POSTGRES_PASSWORD=root
>       - POSTGRES_DB=trading_view_connector
>       - APP_KEY=1234
>       - TELEGRAM_BOT_TOKEN=1234
>       - API_PASSWORD=1234
>       - TELEGRAM_CHAT_ID=1234
>       - TELEGRAM_CHAT_ID2=1234
> ```

Good to know, but I read tip from docker doc
`Don't use environment variables to pass sensitive information, such as passwords, in to your containers. Use [secrets](https://docs.docker.com/compose/use-secrets/) instead.`
Anyway, this is not the reason for my issue

**Comment by LaurentGoderre** [comment]
Yeah I know, I was trying to get the app started to see if I can reproduce. It starts but I didn't configure it correctly so it just gives errors

**Comment by vitassuper** [comment]
> Yeah I know, I was trying to get the app started to see if I can reproduce. It starts but I didn't configure it correctly so it just gives errors

What errors did you get?

**Comment by LaurentGoderre** [comment]
![Screenshot 2023-12-13 at 10 31 40 AM](https://github.com/docker-library/python/assets/1090826/1f3a5aed-d465-495c-b9e5-64fb88decabc)

**Comment by blopker** [comment]
Maybe this is useful, but looking at the repo, flux-bot is using a `slim` variant. This variant strips debug info for file size, where the full fat images do not. Just found this out in https://github.com/docker-library/python/issues/825#issuecomment-1869792567
